pkg/nsx/services/subnet/compare.go

```diff
@@ -0,0 +1,34 @@
+package subnet
+
+import (
+	"github.com/vmware/vsphere-automation-sdk-go/runtime/data"
+	"github.com/vmware/vsphere-automation-sdk-go/services/nsxt/model"
+
+	"github.com/vmware-tanzu/nsx-operator/pkg/nsx/services/common"
+)
+
+type (
+	Subnet     model.VpcSubnet
+	Comparable common.Comparable
```

Comment:
It should be
type Comparable = common.Comparable
because we have no other function added to it.

Reply:
Thanks for reminder, done.